src/components/Logo.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@testing-library/react";
+import Logo from "./Logo";
+
+const expectedIconClasses = "ms-icon icon-doppler-logo";
+
+describe("<Logo />", () => {
+  it("renders logo link", () => {
+    render(<Logo />);
```

Comment:
MINOR... Me parece que hacer un solo render en el _**describe**_ te funciona bien si estamos ante un solo escenario de prueba.
```suggestion
  render(<Logo />);
  it("renders logo link", () => {
```
Removeria tambien el render el otro _**it**_ que esta en este _**describe**_

Reply:
No me funcionó aplicando el cambio sugerido, falla el segundo test si remuevo ese render.